components/brave_ads/browser/ads_service_impl.h

```diff
@@ -32,6 +32,7 @@
 #include "mojo/public/cpp/bindings/associated_receiver.h"
 #include "mojo/public/cpp/bindings/associated_remote.h"
 #include "mojo/public/cpp/bindings/remote.h"
+#include "services/network/public/mojom/url_response_head.mojom.h"
```

Comment:
is there a dep for this already? If not please add

Reply:
Adding in another PR, thanks